distribution/docs/src/main/resources/content/_metadataReference/NITF_Header-table.adoc

```diff
@@ -103,7 +103,7 @@ else current time
 |false
 
 |NitfHeader.FSCLAS
-|none
+|FSCLAS -> ISM classification
```

Comment:
❓Does the documentation for the image attributes need to be updated too?

Reply:
No.

Reply:
No.